extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -78,19 +78,22 @@ amp-carousel .amp-carousel-button.amp-disabled {
 }
 
 .-amp-slides-container {
-  display: flex;
+  align-items: center !important;
+  display: flex !important;
+  flex-wrap: nowrap;
   height: 100% !important;
   overflow-x: auto !important;
   overflow-y: hidden !important;
-  position: relative;
-  white-space: nowrap;
+  position: relative !important;
+  text-align: center;
+  white-space: nowrap !important;
   width: 100% !important;
   scroll-snap-type: mandatory !important;
   -webkit-overflow-scrolling: touch !important;
 }
 
 .-amp-slide-item {
-  display: none;
+  display: none !important;
```

Comment:
One option instead of playing with `!important` here is to simply manage `display` property in JS via `toggle()`. At this point it's up to you, but there might be some reasons why `toggle` might become more convenient in the future.